birthdays/birthdays.py

```diff
@@ -148,8 +151,16 @@ async def bday_list(self, ctx: Context):
                                           for u_id, year in bdays.get(str(date.toordinal()), {}).items())
                               for date in g if len(bdays.get(str(date.toordinal()))) > 0)
             if not value.isspace():  # Only contains whitespace when there's no birthdays in that month
-                embed.add_field(name=datetime.datetime(year=1, month=k, day=1).strftime("%B"), value=value)
-        await message.channel.send(embed=embed)
+                msg += f"{bold(datetime.datetime(year=1, month=k, day=1).strftime('%B'))}\n"
+                msg += f"{value}\n\n"
+
+        for page in pagify(msg, delims=["\n\n"], page_length=2000):
+            embed = discord.Embed(description=page, color=discord.Colour.lighter_grey())
+            embed_list.append(embed)
+
+        for i, em in enumerate(embed_list):
+            em.set_footer(text=f"Page {i + 1}/{len(embed_list)}")
+            await ctx.send(embed=em)
```

Comment:
Instead of paginating into a list and then enumerating on the list, could you combine both of them into one loop? It would eliminate the need for `embed_list` and make this slightly clearer to read I think

If it becomes unreadable you could do something like `pages = pagify(...)` and then `for i, em in enumerate(pages)`